app/controllers/api/v1/occupation_standards_controller.rb

```diff
@@ -5,9 +5,9 @@ class API::V1::OccupationStandardsController < API::V1::APIController
 
   def index
     @oss = OccupationStandard.with_eager_loading
-             .search(search_params.to_h)
+             .search_records(search_params.to_h)
              .where.not(type: "UnregisteredStandard")
-             .order(id: :desc)
+             .order(id: :asc)
```

Comment:
Looks like the `where.not` needs to be moved before the `search_records` since `search_records` returns a `Searchkick::Result` object. Also, do we need to move the ordering as part of the searchkick query?